src/Features/Core/Portable/ChangeSignature/AbstractChangeSignatureService.cs

```diff
@@ -314,10 +316,10 @@ internal ChangeSignatureResult ChangeSignatureWithContext(ChangeSignatureAnalyze
 
             if (hasLocationsInMetadata)
             {
-                var notificationService = context.Solution.Workspace.Services.GetService<INotificationService>();
+                var notificationService = context.Solution.Workspace.Services.GetService<INotificationService>()!;
```

Comment:
can we add a `.RequireService` that throws if we can't find the service.  We're going to blow up on the next line anyways.

Reply:
Seems there's already a method like that.

Reply:
yaay :)